content/en/docs/tasks/observability/distributed-tracing/configurability/index.md

```diff
@@ -8,13 +8,49 @@ test: no
 ---
 
 Istio provides the ability to configure advanced tracing options,
-including sampling rates and span tags. Sampling is a beta feature, but
-custom tags and tracing tag length are considered in-development for this release.
+such as sampling rate and adding custom tags to reported spans.
+Sampling is a beta feature, but adding custom tags and tracing tag length are
+considered in-development for this release.
 
-## Create a `MeshConfig` with trace settings
+## Before you begin
 
-All tracing options are configured by using `MeshConfig` during Istio *installation*.
-To simplify configuration, create a single YAML file to pass to `istioctl`.
+1.  Follow the tracing installation guide located under [Integrations](/docs/ops/integrations/)
+    based on your preferred tracing backend to install the appropriate addon and
+    configure your Istio proxies to send traces to the tracing deployment.
+
+1.  Ensure that your applications propagate tracing headers as described [here](/docs/tasks/observability/distributed-tracing/overview/).
```

Comment:
i wonder if we should switch the ordering here. Updating apps to propagate context is probably the first thing that should be done.

Reply:
make sense, will switch it.